MultimodalQnA/multimodalqna.py

```diff
@@ -333,8 +351,12 @@ async def handle_request(self, request: Request):
                 return response
         last_node = runtime_graph.all_leaves()[-1]
 
+        tts_audio = ""
         if "text" in result_dict[last_node].keys():
             response = result_dict[last_node]["text"]
+            # Toggle for TTS
+            if "audio" in data["modalities"]:
```

Comment:
```suggestion
            if "modalities" in data and "audio" in data["modalities"]:
```

Reply:
Actually, after you make the change that I posted in GenAIComps, can you try changing this to use `chat_request.modalities` instead of the `data` dict?